src/arr/compiler/pyret.arr

```diff
@@ -89,15 +93,16 @@ fun main(args):
         B.set-allow-builtin-overrides(r.get-value("allow-builtin-overrides"))
       end
       if not(is-empty(rest)):
-        raise("No longer supported")
+        _ = print("No longer supported\n")
```

Comment:
I would use `block:` and `print-error` here.  Normally not this nitpicky, but I'd like to move towards `pyret.arr` being an exemplar of good style.